core/controllers/story_viewer.py

```diff
@@ -78,7 +78,8 @@ def get(self, story_id):
             'story_title': story.title,
             'story_description': story.description,
             'story_nodes': ordered_node_dicts,
-            'topic_name': topic_name
+            'topic_name': topic_name,
+            'meta_tag_content': story.meta_tag_content
```

Comment:
Shouldn't there be tests as well for this change? Ditto below.

Reply:
I've added this field in the relevant test too. Done.